salesforce/tests/test.bal

```diff
@@ -213,6 +213,45 @@ function testQueryWithLimit() returns error? {
     }
 }
 
+type AccountResultWithAlias record {|
+    string Name;
+    int NumAccounts;
+|};
```

Comment:
In Ballerina we usually define the type definitions in a separate section (either at the top of the same file or, in a `types.bal` file). Shall we follow the same convention in here?